services/matrix/synapse-log_config.yaml

```diff
@@ -0,0 +1,27 @@
+# Based on https://github.com/NixOS/nixpkgs/blob/nixos-unstable/nixos/modules/services/matrix/synapse-log_config.yaml
+# Just with a log-level of notice
```

Comment:
Correct that

Reply:
This is still notice?

Reply:
Finally corrected now ;D Thinking that one has changed something is seemingly not the same as actually changing it...